apps/condo/migrations/20210720124515-0027_organizationlink_organizationlinkhistoryrecord.js

```diff
@@ -0,0 +1,46 @@
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- Create model organizationlinkhistoryrecord
+--
+CREATE TABLE "OrganizationLinkHistoryRecord" ("dv" integer NULL, "sender" jsonb NULL, "from" uuid NULL, "to" uuid NULL, "id" uuid NOT NULL PRIMARY KEY, "v" integer NULL, "createdAt" timestamp with time zone NULL, "updatedAt" timestamp with time zone NULL, "createdBy" uuid NULL, "updatedBy" uuid NULL, "deletedAt" timestamp with time zone NULL, "newId" jsonb NULL, "history_date" timestamp with time zone NOT NULL, "history_action" varchar(50) NOT NULL, "history_id" uuid NOT NULL);
+--
+-- Create model organizationlink
+--
+CREATE TABLE "OrganizationLink" ("dv" integer NOT NULL, "sender" jsonb NOT NULL, "id" uuid NOT NULL PRIMARY KEY, "v" integer NOT NULL, "createdAt" timestamp with time zone NULL, "updatedAt" timestamp with time zone NULL, "deletedAt" timestamp with time zone NULL, "newId" uuid NULL, "createdBy" uuid NULL, "from" uuid NOT NULL, "to" uuid NOT NULL, "updatedBy" uuid NULL);
+CREATE INDEX "OrganizationLinkHistoryRecord_history_id_57900a37" ON "OrganizationLinkHistoryRecord" ("history_id");
+ALTER TABLE "OrganizationLink" ADD CONSTRAINT "OrganizationLink_createdBy_62294a28_fk_User_id" FOREIGN KEY ("createdBy") REFERENCES "User" ("id") DEFERRABLE INITIALLY DEFERRED;
+ALTER TABLE "OrganizationLink" ADD CONSTRAINT "OrganizationLink_from_8684a1b6_fk_Organization_id" FOREIGN KEY ("from") REFERENCES "Organization" ("id") DEFERRABLE INITIALLY DEFERRED;
+ALTER TABLE "OrganizationLink" ADD CONSTRAINT "OrganizationLink_to_7b976588_fk_Organization_id" FOREIGN KEY ("to") REFERENCES "Organization" ("id") DEFERRABLE INITIALLY DEFERRED;
+ALTER TABLE "OrganizationLink" ADD CONSTRAINT "OrganizationLink_updatedBy_4406935e_fk_User_id" FOREIGN KEY ("updatedBy") REFERENCES "User" ("id") DEFERRABLE INITIALLY DEFERRED;
+CREATE INDEX "OrganizationLink_createdAt_1881ba3b" ON "OrganizationLink" ("createdAt");
+CREATE INDEX "OrganizationLink_updatedAt_a84564af" ON "OrganizationLink" ("updatedAt");
+CREATE INDEX "OrganizationLink_deletedAt_3c8dfca4" ON "OrganizationLink" ("deletedAt");
+CREATE INDEX "OrganizationLink_createdBy_62294a28" ON "OrganizationLink" ("createdBy");
+CREATE INDEX "OrganizationLink_from_8684a1b6" ON "OrganizationLink" ("from");
```

Comment:
Please, rebase with what @dkoviazin made in #319, — he had also committed migration 0027.
You should have 0028.
I have 0028 and it depends, who will merge it's pull-request first.

Reply:
Why should I rebase from this branch? We have 0027 in the master. If I delete my migration files, then rebase from the master and do `makemigrations` and` maketypes` again, won't I get the desired result?

Reply:
I meant, that you should rebase to see others changes.
Yes, `makemigrations` will recreate your migrations, after you delete your migration file first.

Reply:
ok, I created a 0028 migration in dae0314, which comes after 0027 from the master branch.